doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,188 @@
+- Candidate CNF Validation will ensure interoperablity of CNF behaviour on the Kubernetes infrastructure.  Testing ensures CNFs can be spun up, modified, or removed, on the target infrastructure (aka Interoperability).  
+
+<a name="1.1.1"></a>
+### 1.1.1 Terminology
+
+Terminology in this document will follow [CNTT Terminology](../../../tech/glossary.md). Relevant testing terminology includes:
+
+<a name="1.2"></a>
+## 1.2 Scope
+
+This document covers aspects of CNTT conformance for both Kubernetes based cloud infrastructure and CNFs. The document will cover the following topics:
+
+- Identify in details the requirements of test-cases (mapped from RA2 and RM)
+- Test criteria that shows a certain capability or feature of the system-under-test exists and behaves as expected
+- An E2E framework for conformance of Kubernetes infrastructures and CNFs, including specification for conformance test infrastructure (lab environment and tools)
+- Analysis to identify where the gaps are in the industry for implementing conformance test objectives (tooling, methods, process, etc).
+
+**Not in Scope**
+- Functional testing / validation of the application provided by the CNF is outside the scope of this work.
+- Testing to confirm specific versions of APIs or software components is out of scope of conformance since this is an unambiguous part of a product specification.
+- VNFM/NFVO, like ONAP, is not used in the process flow for infrastructure verifications, or validations.
+- Upgrades to CNFs, and the respective processes of verifying upgrade procedures and validating (testing) the success and compatibility of upgrades is not in scope.
+
+<a name="1.3"></a>
+## 1.3 Principles and Guidelines
+
+The objectives of the Reference Conformance for cloud infrastrucute is to verify implementations against the reference architecture which satisfies infrastructure needs for CNFs.
+
+The objectives of the Reference Conformance for CNFs is to verify CNF implementations consume resources and behave as expected against the reference architecture.
+
+These core principles will guide RC2 deliverables:
+
+- RC-2 requirements are derived from RM and RA-2 which specify infrastructure capabilities which include compute, memory, storage resource capabilities, performance optimization capabilities, monitoring capabilities.
+
+- Requirements in the RM and RAs that are performance related may not have minimum perfromance criteria identified but where feasible will have tests with metrics to show relevant capabilities are present and working as expected.
+
+- Must/shall conformance criteria are testable as pass/fail and ensure requirement are met to ensure minimum thresholds of functional operability and/or performance behavior. This is the focus of RC-2 since this is what will drive a commercially significant badging program.
```

Comment:
```suggestion
- Must/shall conformance criteria are testable as pass/fail and ensure requirements are met to ensure minimum thresholds of functional operability and/or performance behavior. This is the focus of RC-2 since this is what will drive a commercially significant badging program.
```